cmd/installOprerator_test.go

```diff
@@ -0,0 +1,37 @@
+package cmd
```

Comment:
This file needs to be renamed (`s/Oprerator/Operator/`).

Reply:
Forgot to delete this file @tumido  ?

Reply:
I can't rebase properly today... :stuck_out_tongue: Fixed, thanks :slightly_smiling_face: